packages/lit-dev-content/samples/checkable-tutorial-base.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "/samples/base.json",
+  "files": {
+    "_check-code-helpers.js": {
```

Comment:
Suggest making this an actual `.ts` file instead of inlining. All files in a `project.json` are automatically converted to be inline by https://github.com/lit/lit.dev/blob/main/packages/lit-dev-tools-cjs/src/playground-inline.ts.

Reply:
TIL!